frontends/web/src/locales/en/app.json

```diff
@@ -197,6 +197,40 @@
   "bitboxBase": {
     "connect": "Connect Base",
     "detectedBases": "Detected the following devices on your network:",
+    "errors": {
+      "BACKUP_SYSCONFIG_NOT_A_MOUNTPOINT": "Internal error: {{error}}",
```

Comment:
yo urepeat "Internal error:" a lot here. it would be good to just name that `errors.internal` and use this in its place.

Reason: every entry here costs money in terms of the locize price model as well as the translators translating each entry into every language.

Reply:
The errors are originating in the Middleware and I don't want to compromise on their granularity. How could we handle that in the App to only have one single "internal error" entry? @Tomasvrba

Reply:
Replaced the "Internal error" with a variable, as discussed with @Tomasvrba